web-common/src/features/dashboard-stores.spec.ts

```diff
@@ -136,4 +144,88 @@ describe("dashboard-stores", () => {
       AdBidsBidPriceMeasure
     );
   });
+
+  describe("Restore invalid state", () => {
+    it("Restore invalid filter", () => {
+      createAdBidsInStore();
+      metricsExplorerStore.toggleFilter(
+        AdBidsName,
```

Comment:
This should be an all-caps constant. And a more descriptive name could be `ADBIDS_DASHBOARD_NAME`.